components/data-table/column.jsx

```diff
@@ -41,7 +41,6 @@ DataTableColumn.propTypes = {
 	 * </DataTable>
 	 * ```
 	 */
-	children: PropTypes.element,
```

Comment:
This should be kept, due to how DataTables work. Please see documentation https://react.lightningdesignsystem.com/components/data-table/